packages/xo-web/src/xo-app/hub/resource.js

```diff
@@ -171,10 +180,72 @@ export default decorate([
       redirectToTaskPage() {
         this.props.router.push('/tasks')
       },
+      showDescription() {
+        const {
+          data: { public: _public },
+          name,
+        } = this.props
+        alert(
+          name,
+          <div>
+            {isEmpty(omit(_public, BANNED_FIELDS)) ? (
+              <p>{_('hubTemplateDescriptionNotAvailable')}</p>
+            ) : (
+              <div>
+                <Ul>
+                  {EXCLUSIVE_FIELDS.map(fieldKey => {
+                    const field = _public[fieldKey]
+                    return field !== undefined ? (
+                      <Li key={fieldKey}>
+                        <div
+                          dangerouslySetInnerHTML={{
+                            __html: marked(field),
+                          }}
+                        />
+                      </Li>
+                    ) : null
+                  })}
+                </Ul>
+                <br />
+                <Ul>
+                  {map(omit(_public, STATIC_FIELDS), (value, key) => (
+                    <Li key={key}>
+                      {startCase(key)}
+                      <span className='pull-right'>
+                        {typeof value === 'boolean' ? (
+                          <Icon
+                            color={value ? 'green' : 'red'}
+                            icon={value ? 'success' : 'close'}
```

Comment:
`'close'` is weird since it's not an icon used to close something. Don't hesitate to create generic `'true'` and `'false'` icons or something like that if they don't already exist. If they're very clear and reusable, it's not a problem to create new ones.